packages/gatsby-remark-images/README.md

```diff
@@ -52,7 +52,7 @@ plugins: [
 | `sizeByPixelDensity`   | `false` | Analyze images' pixel density to make decisions about target image size. This is what GitHub is doing when embedding images in tickets. This is a useful setting for documentation pages with a lot of screenshots. It can have unintended side effects on high pixel density artworks.<br /><br />Example: A screenshot made on a retina screen with a resolution of 144 (e.g. Macbook) and a width of 100px, will be rendered at 50px.                                                                                                                                                                                           |
 | `wrapperStyle`         |         | Add custom styles to the div wrapping the responsive images. Use the syntax for the style attribute e.g. `margin-bottom:10px; background: red;` or a function returning a style string which receives the information about the image you can use to dynamically set styles based on the aspectRatio for example.                                                                                                                                                                                                                                                                                                                  |
-| `backgroundColor`      | `white` | Set the background color of the image to match the background image of your design.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `backgroundColor`      | `white` | Set the box-shadow color of the image to match the background image of your design.<br /><br />**Note:** To completely disable box-shadow, use `backgroundColor: none`. You can also use `backgroundColor: transparent` for a transparent box-shadow (i.e., `box-shadow: inset 0px 0px 0px 400px transparent`).                                                                                                                                                                                                                                                                                                                    |
```

Comment:
There are 2 separate things here:
 - resulting effect (which is background after all)
 - implementation (which uses box-shadow)

Just specifying one of those will always confuse someone, so maybe adjusting this to somehow use `background` to describe effect and `box-shadow` to describe implementation will make it so users looking for `box-shadow` will still find this and people just looking at prop list will not be confused?

Reply:
hi, my main reason for mentioning `backgroundColor: 'transparent'` and `backgroundColor: 'none'` was because I wanted to remove `box-shadow` entirely from `<img class="gatsby-resp-image-image" />`. I was a bit confused with the `backgroundColor` option because I thought I can only change the box-shadow color, but not remove it. After I changed from `backgroundColor: 'transparent'` to `backgroundColor: 'none'`, I noticed that this would remove the `box-shadow` from the `<img />` element. I thought mentioning this in the doc might help someone else, but I don't want this to confuse anyone. Maybe we should just close this PR. I can work on something else instead.

Reply:
Actually, maybe we can remove the changes to README, but still keep the unit tests